lib/api/v1/user-controller.js

```diff
@@ -66,10 +70,12 @@ module.exports = function(app, user) {
   });
 
   // Update a User
-  app.put('/api/v1/user/:_id', user.can('manage users'), function(req, res) {
+  app.put('/api/v1/user/:_id', user.can('update users'), function(req, res) {
     User.findById(req.params._id, function(err, user) {
       if (err) return res.send(err.status, err.message);
       if (!user) return res.send(404);
+      //Only admin can update users other than itself
+      if (req.user && req.user.role !== 'admin' && req.params._id != req.user._id) return res.send(403);
```

Comment:
Similarly, this should have a permission attached to it. `admin users` is the right one, I think?